content/latest/faq/compatibility.md

```diff
@@ -43,6 +45,22 @@ YugaByte DB's goal is to remain as compatible with PostgreSQL as possible. If yo
 
 ## YCQL Compatibility with Apache Cassandra QL
 
+### List the features where YCQL goes beyond Apache Cassandra QL.
+
+- Following are the features that are present in YCQL but not present in Apache Cassandra QL.
+
+1. [JSONB](../../develop/learn/data-types/) column type for modeling document data
```

Comment:
Maybe also worth calling out these:

* The INTEGER type in CQL cannot be incremented, you need to use the COUNTER type for that. Increment operations need LWT which is 4 roundtrips. No difference between these two types in YugaByte DB - both can be assigned or incremented, and increment operations are very fast with only one round trip.

* Indexes on dynamic attributed in JSONB types are being worked on.

* Online backfill of indexes after data is loaded is being worked.

Reply:
Also, YCQL passed Jepsen testing for transactional use-cases while other CQL databases do not.